File: src/main/scala/org/openapitools/server/model/CalculateTariffsResponseDTO.scala
package org.openapitools.server.model


/**
 * Расчет стоимости услуг.
 *
 * @param offers Стоимость услуг. for example: ''null''
*/
final case class CalculateTariffsResponseDTO (
  offers: Seq[CalculateTariffsOfferInfoDTO]
)

File: ympa_fsharp_functions_server/src/model/CalculateTariffsResponseDTO.fs
namespace ympa_fsharp_functions_server.Model

open System
open System.Collections.Generic
open Newtonsoft.Json
open ympa_fsharp_functions_server.Model.CalculateTariffsOfferInfoDTO

module CalculateTariffsResponseDTO =

  //#region CalculateTariffsResponseDTO

  [<CLIMutable>]
  type CalculateTariffsResponseDTO = {
    [<JsonProperty(PropertyName = "offers")>]
    Offers : CalculateTariffsOfferInfoDTO[];
  }

  //#endregion
